dynamiq/nodes/node.py

```diff
@@ -225,6 +227,13 @@ def __init__(self, **kwargs):
 
         self._output_references = NodeOutputReferences(node=self)
 
+    @field_validator("name")
+    def sanitize_name(s: str):
```

Comment:
Do we have any issues with naming? E.g I would to use `[Andrii] OpenAI` name. Any reason why I can't?

Reply:
Yes because:

1) On UI we can only provide name in this format
2) Function calling require this format.
3) It transforms to appropriate format automatically.
4) I guess it is better to keep names in this format, easier to read in traces.